pkg/storaged/crypto-keyslots.jsx

```diff
@@ -692,21 +690,19 @@ const RemovePassphraseField = (tag, key, dev) => {
             return (
                 <Stack hasGutter>
                     <p>{ fmt_to_fragments(_("Passphrase removal may prevent unlocking $0."), <b>{dev}</b>) }</p>
-                    <Form>
-                        <Checkbox id="force-remove-passphrase"
-                                  isChecked={val !== false}
-                                  label={_("Confirm removal with an alternate passphrase")}
-                                  onChange={(_event, checked) => change(checked ? "" : false)}
-                                  body={val === false
-                                      ? <p className="slot-warning">
-                                          {_("Removing a passphrase without confirmation of another passphrase may prevent unlocking or key management, if other passphrases are forgotten or lost.")}
-                                      </p>
-                                      : <FormGroup label={_("Passphrase from any other key slot")} fieldId="remove-passphrase">
-                                          <TextInputPF id="remove-passphrase" type="password" value={val} onChange={(_event, value) => change(value)} />
-                                      </FormGroup>
-                                  }
-                        />
-                    </Form>
+                    <Checkbox id="force-remove-passphrase"
+                                isChecked={val !== false}
+                                label={_("Confirm removal with an alternate passphrase")}
+                                onChange={(_event, checked) => change(checked ? "" : false)}
+                                body={val === false
+                                    ? <p className="slot-warning">
+                                        {_("Removing a passphrase without confirmation of another passphrase may prevent unlocking or key management, if other passphrases are forgotten or lost.")}
+                                    </p>
+                                    : <FormGroup label={_("Passphrase from any other key slot")} fieldId="remove-passphrase">
+                                        <TextInputPF id="remove-passphrase" type="password" value={val} onChange={(_event, value) => change(value)} />
+                                    </FormGroup>
```

Comment:
These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102432-6e7b9faf-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)